README.md

```diff
@@ -17,7 +17,7 @@ For the latest news and updates, follow [@FacesBridgeSpec](https://twitter.com/F
 
 Using [Maven](https://maven.apache.org/) 3.x:
 
-	mvn clean install
+    mvn clean install
```

Comment:
@ngriffin7a, not particularly. I just figured since I was changing so much, I should make the formatting consistent. Most indentations were spaces as far as I could see. So I just replaced the tabs with spaces.